packages/fluentui/react-northstar/src/components/Avatar/Avatar.tsx

```diff
@@ -129,12 +132,14 @@ export const Avatar: ComponentWithAs<'div', AvatarProps> & FluentComponentStatic
     <ElementType {...getA11Props('root', { className: classes.root, ...unhandledProps })}>
       {hasGlyph && (imageElement || iconElement)}
       {!hasGlyph && labelElement}
-      {Status.create(status, {
+      {createShorthand(AvatarStatus, status, {
         defaultProps: () =>
           getA11Props('status', {
             size,
-            styles: resolvedStyles.status,
```

Comment:
As discussed offline with @layershifter  let's keep passing this down and mark for deprecation, so it's one less breaking change to worry about. We can create a ticket to move the slot style for TMP so that adoption is easier